ext/symengine/symengine.c

```diff
@@ -46,4 +47,10 @@ void Init_symengine() {
     c_rational = rb_define_class_under(m_symengine, "Rational", c_basic);
     rb_define_alloc_func(c_rational, cbasic_alloc);
     rb_define_method(c_rational, "initialize", crational_init, 1);
+
+    //Constants
+    c_constants = rb_define_class_under(m_symengine, "Constants", c_basic);
+    rb_define_const(m_symengine, "PI", cconstants_pi());
+    rb_define_const(m_symengine, "E", cconstants_e());
+    rb_define_const(m_symengine, "EulerGamma", cconstants_euler_gamma());
```

Comment:
I was using the notation used at [SymEngine CWrapper](https://github.com/symengine/symengine/blob/master/symengine/cwrapper.cpp#L106). Should I go ahead and change to All Caps or leave it as it is.